### ISSUE TITLE
Better storage format - go JSON?

### ISSUE THREAD

**Issue by MansMeg** [MansMeg/SwedishPolls#153]
Now we use a CSV as a poll. But in the long run we would like to have more information on individual polls, so we should try a JSON storage format instead, where additional data can be read easily and more flexible in adding new information. Then have helper functions to access the data.

Any takes here @StaffanBetner and @hampusborgos ?

**Comment by StaffanBetner** [comment]
I agree. One JSON file per poll is probably the best approach. Makes it easier for historical data as well.

**Comment by StaffanBetner** [comment]
Some additional details that could be recorded: the polling method (telephone interviews, panels, etc.) and the precision, meaning the smallest reported unit. For example, Ipsos uses 1 percentage point, many older polls used 0.5, and most others use 0.1.
